src/gcp/artifactregistry.ts

```diff
@@ -1,14 +1,32 @@
 import { Client } from "../apiv2";
 import { artifactRegistryDomain } from "../api";
+import { needProjectId } from "../projectUtils";
+import * as api from "../ensureApiEnabled";
+import * as proto from "./proto";
 
-export const API_VERSION = "v1beta2";
+export const API_VERSION = "v1";
 
 const client = new Client({
   urlPrefix: artifactRegistryDomain(),
   auth: true,
   apiVersion: API_VERSION,
 });
 
+export function ensureApiEnabled(options: any): Promise<void> {
```

Comment:
nit: It would be cleaner if we just passed a projectId rather than propagating an untyped object.